java/src/main/kotlin/io/spine/protodata/java/FieldAccess.kt

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2024, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.protodata.java
+
+import com.google.protobuf.Message
+import io.spine.protodata.ast.Cardinality
+import io.spine.protodata.ast.FieldName
+import io.spine.protodata.ast.fieldName
+
+/**
+ * A selector to access methods for a Protobuf message field.
+ *
+ * Depending on the field type, it may allow generating getters and setters for the field.
+ */
+public class FieldAccess
+internal constructor(
+    public val message: Expression<Message>,
+    name: FieldName,
+    kind: Cardinality,
+) : FieldConventions(name, kind) {
+
+    /**
+     * Constructs field access for the given [message] and [name].
+     */
+    internal constructor(
+        message: Expression<Message>,
+        name: String,
+        kind: Cardinality = Cardinality.CARDINALITY_SINGLE
```

Comment:
Please import enum items. They all have the prefix, so we'd know what they are about.